src/status_im/contexts/chat/messenger/messages/content/emoji/view.cljs

```diff
@@ -0,0 +1,10 @@
+(ns status-im.contexts.chat.messenger.messages.content.emoji.view
+  (:require [react-native.core :as rn]
+            [status-im.contexts.chat.messenger.messages.content.emoji.style :as style]))
+
+(defn emoji-message
+  [{:keys [content last-in-group? pinned]} {:keys [in-pinned-view?]}]
```

Comment:
for consistency. Other message components pass both the message data and message context as separate maps. It would be better to only pass the subset of props that it needs, i agree

Reply:
I think it makes sense to only pass the parameters it needs (positional probably faster, but less explicit), it's very static content, so we won't have to change it often